signac/__main__.py

```diff
@@ -522,7 +522,7 @@ def _sig(st):
             exclude=args.exclude,
             doc_sync=doc_sync,
             selection=selection,
-            check_schema=not args.force,
+            check_schema=not args.no_check_schema,
```

Comment:
I am sorry, I should have been more clear about this. Please do not replace `-f`/`--force` option, but add an additional one.

Removing an existing option is not a backwards-compatible change and the `-f`/`--force` option is used to override other checks as well.